lighthouse-core/runner.js

```diff
@@ -137,6 +146,7 @@ class Runner {
           initialUrl: opts.initialUrl,
           url: opts.url,
           audits: formattedAudits,
+          traces: opts.includeTrace ? tracesToInclude : undefined,
```

Comment:
If you init'd `let tracesToInclude;` then you could just do: `traces: tracesToInclude` here.